platform/ios/src/MGLMapView.h

```diff
@@ -251,7 +251,8 @@ MGL_EXPORT IB_DESIGNABLE
 
 /**
  A view showing legally required copyright notices and telemetry settings,
- positioned at the bottom-right of the map view.
+ positioned at the bottom-right of the map view. Tapping on this button will
+ call `-showAttribution`.
 
  @note The Mapbox terms of service, which governs the use of Mapbox-hosted
     vector tiles and styles,
```

Comment:
This is a better place to mention the new action, probably.

Reply:
This new action is equally relevant to both the attribution and telemetry `@note` sections, though. How about something like:

> A view showing legally required copyright notices and telemetry settings,
 positioned at the bottom-right of the map view.
>
> If you choose to reimplement this view, assign the `-showAttribution:` method
 as the action for your view to present the default notices and settings.
>
> `@note` ...